### DISCUSSION TITLE
Scatter Plot Chart

### DISCUSSION THREAD

**Discussion by mraustinmiller** [taiga-family/taiga-ui#10074] [Ideas]
I am creating some data visualization applications and it would be greatly beneficial if Taiga had a scatter plot chart. 
Possible inclusions:
- tooltips
- single/cluster selection
- (Moving) Average

I can use other chart api's, but with already using Taiga it would be easier to still to less dependencies.

**Comment by waterplea** [comment]
At the moment Taiga UI charts are not our top priority but I passed along this request to our designers.
